### PR TITLE
CCIP-4014 enable rmn tests in ci

### PR DESCRIPTION
**Pull request by AnieeG**

### CODE REVIEW

**Review by makramkd — DISMISSED** [review]

File: .github/e2e-tests.yml [hunk 1]
```diff
@@ -991,6 +991,104 @@ runner-test-matrix:
...
+      - Nightly E2E Tests
+    test_cmd: cd integration-tests/smoke && go test -test.run ^TestRMN_DifferentSigners$ -timeout 12m -test.parallel=1 -count=1 -json
+    pyroscope_env: ci-smoke-ccipv1_6-evm-simulated
+    test_env_vars:
+      E2E_TEST_SELECTED_NETWORK: SIMULATED_1,SIMULATED_2
+      E2E_JD_VERSION: 0.4.0
+      E2E_RMN_RAGEPROXY_VERSION: master-5208d09
+      E2E_RMN_AFN2PROXY_VERSION: master-5208d09
+
+#  - id: smoke/ccip_rmn_test.go:^TestRMN_NotEnoughSigners$
```
Comment (mateusz-sekara) [inline .github/e2e-tests.yml:1060]: I see a test like this in the repo, why do we comment it out?

File: integration-tests/smoke/ccip_rmn_test.go [hunk 2]
```diff
@@ -26,10 +27,8 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
-// Set false to run the RMN tests
-const skipRmnTest = true
-
 func TestRMN_TwoMessagesOnTwoLanes(t *testing.T) {
+	t.Parallel()
```
Comment (mateusz-sekara) [inline integration-tests/smoke/ccip_rmn_test.go]: Do we need `t.Parallel()` if every test is run as a separate CI job?

File: integration-tests/testconfig/ccip/config.go [hunk 3]
```diff
@@ -45,6 +49,38 @@ type RMNConfig struct {
...
+
+func (r *RMNConfig) GetAFNImage() string {
+	image := pointer.GetString(r.AFNImage)
+	if image == "" {
+		return ctfconfig.MustReadEnvVar_String(E2E_RMN_AFN2PROXY_IMAGE)
+	}
+	return image
+}
+
+func (r *RMNConfig) GetAFNVersion() string {
```
Comment (mateusz-sekara) [inline integration-tests/testconfig/ccip/config.go]: Shouldn't we align with a new naming? I guess no one remembers what AFN was :P

**Review by makramkd — DISMISSED** [review]

File: .github/e2e-tests.yml (hunk 1 above)
Comment (AnieeG) [reply .github/e2e-tests.yml:1060]: This is currently failing @dimkouv is working on a PR to fix this. After that we will uncomment this

File: integration-tests/smoke/ccip_rmn_test.go (hunk 2 above)
Comment (AnieeG) [reply integration-tests/smoke/ccip_rmn_test.go]: Tried parallel at first did not work. can remove it

File: integration-tests/testconfig/ccip/config.go (hunk 3 above)
Comment (AnieeG) [reply integration-tests/testconfig/ccip/config.go]: The image name is AFN2Proxy https://github.com/smartcontractkit/rmn-v2-test-infra/tree/master/docker/afn2proxy

**Review by mateusz-sekara — DISMISSED** [review]

**Review by mateusz-sekara — APPROVED** [review]

**Review by makramkd — APPROVED** [review]

**Pull request merged**